### ISSUE TITLE
IllegalArgumentException: location.world

### ISSUE THREAD

**Issue by mibby** [Efnilite/Walk-in-the-Park#17]
Compiled as of https://github.com/Efnilite/Walk-in-the-Park/commit/2f21af98ebc02d0df37b80ae981382070c839566

* Player joins WITP.
* Used options to quit.
* It didn't bring them to spawn. They were falling in the void suffocating.
* Since they were no longer part of WITP, they could use ``/spawn`` to leave.
* However they lost their inventory and retained the clock.

I'm assuming this is due to ``go-back-enabled: true`` being enabled. Setting it false fixes the issue.

```
[09:09:50] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WITP v2.0
[09:09:50] [Server thread/ERROR]: Suspected Plugins: 
[09:09:50] [Server thread/ERROR]: 	WITP{enabled,ver=2.0,path=plugins/WITP-2.1.jar}
[09:09:50] [Server thread/ERROR]: Exception details below: 
java.lang.IllegalArgumentException: location.world
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:122) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer.teleport(CraftPlayer.java:780) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at org.bukkit.craftbukkit.v1_16_R3.entity.CraftEntity.teleport(CraftEntity.java:515) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at dev.efnilite.witp.player.ParkourUser.unregister(ParkourUser.java:104) ~[?:?]
	at dev.efnilite.witp.player.ParkourPlayer.lambda$menu$17(ParkourPlayer.java:402) ~[?:?]
	at dev.efnilite.witp.util.inventory.InventoryBuilder$ClickHandler.onClick(InventoryBuilder.java:168) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor362.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:611) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2643) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:43) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1218) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1211) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1187) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1027) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:176) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-"a1c007b"]
	at java.lang.Thread.run(Thread.java:832) [?:?]
```

```
# Options about Bungeecord/autojoining
bungeecord:

    # Enabling this will make players who join the server (that's running this plugin) automatically join the game on that server
    # !! If this is disabled, you have to use /witp join to join and /witp leave to leave. !!
    enabled: false

    # The server the players will be returned to after the game is done
    return_server: Lobby

    # If go back is enabled
    go-back-enabled: true

    # The place players will go back to if they leave, if bungeecord is disabled and the go-back option is enabled
    # You need to retain this format, or else it won't work: '(x,y,z,world)'
    go-back: (9,58,0,spawn)

# Options for the world
world:
    # The world in which the personal parkours will take place
    name: witp

    # The world players will be teleported to if they accidentally join the WITP world without being a player
    fall-back: spawn
```

**Comment by Efnilite** [comment]
Using your location works fine for me, if the latest commit doesn't work you can comment again.

**Comment by Efnilite** [comment]
This has been fixed in the latest release (v2.1a) - sorry for it taking a while
